src/pages/Market/styles.ts

```diff
@@ -0,0 +1,74 @@
+import { css } from '@emotion/react';
+import { useTheme } from '@mui/material';
+
+export const useStyles = () => {
+  const theme = useTheme();
+  return {
+    headerRoot: css`
+      padding: ${theme.spacing(4)};
```

Comment:
In the designs, on desktop only the left padding seems to be 24px

Reply:
The spacing between the content and edge of the paper is split between the content and the content container in order to achieve the correct height for the vertical rule

Reply:
I'm talking about this padding:
<img width="225" alt="Screen Shot 2022-05-11 at 16 07 32" src="https://user-images.githubusercontent.com/44675210/167883826-df830b87-0275-4927-a1a8-cbf65654b36e.png">

In the designs it's 24px on desktop, in your code it's 16px.